File: src/KnightFrank.Antares.Dal/Model/Tenancy/TenancyContact.cs
namespace KnightFrank.Antares.Dal.Model.Tenancy
{
    using System;

    using KnightFrank.Antares.Dal.Model.Contacts;
    using KnightFrank.Antares.Dal.Model.Enum;

    public class TenancyContact : BaseEntity
    {
        public Guid TenancyId { get; set; }
        public Guid ContactId { get; set; }
        public virtual Contact Contact { get; set; }
        public Guid ContactTypeId { get; set; }
        public virtual EnumTypeItem ContactType { get; set; }
    }
}

File: src/wwwroot/app/common/models/business/tenancy/tenancyContact.ts
/// <reference path="../../../../typings/_all.d.ts" />

module Antares.Common.Models.Business {
    export class TenancyContact {
        id: string = null;
        contactId: string = null;
        contact: Contact = null;
        contactType: EnumTypeItem = null;
        tenancyId: string = null;
    }
}